utils/https.h

```diff
@@ -19,6 +21,22 @@ extern "C" {
  * @file utils/https.h
  */
 
+/* struct type of HTTP(S) response */
+typedef struct {
+  char* buffer; /**< Message body **/
+  size_t len;   /**< Length of message body */
+} https_response_t;
+
+/* struct type of HTTP(S) context */
```

Comment:
Doxygen may not be able to dump this object correctly. Can you check it out?

Reply:
The struct can be appeared.